ERCS/erc-7655.md

```diff
@@ -0,0 +1,113 @@
+---
+eip: 7655
+title: ERC-5189 Operation Receipts
+description: A general interface for the detection of ERC-5189 operation receipts.
+author: William Hua (@attente), Agustín Aguilar (@agusx1211)
+discussions-to: https://ethereum-magicians.org/t/erc-7655-erc-5189-operation-receipts/19187
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-12
+requires: 5189
+---
+
+## Abstract
+
+This ERC is intended to address the issue of obtaining operation receipts in [ERC-5189](./eip-5189.md), which is an application layer account abstraction proposal that maintains compatibility with existing smart contract wallets.
```

Comment:
Your abstract is a bit too terse. It doesn't give the reader any technical information about how this proposal accomplishes its goal.